google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -110,16 +111,22 @@ JLabel getWarningMessage() {
   }
 
   public boolean isModified() {
-    return !Comparing.strEqual(getCloudSdkDirectory(),
-        CloudSdkService.getInstance().getCloudSdkHomePath());
+    if (getCloudSdkDirectory() != null) {
+      return Paths.get(getCloudSdkDirectory())
+          .compareTo(CloudSdkService.getInstance().getCloudSdkHomePath()) != 0;
+    }
+
+    return CloudSdkService.getInstance().getCloudSdkHomePath() != null;
```

Comment:
After trying this with SDK as null and got an NPE.

Changing so that comparison is made if both are not null, otherwise returns true if any of them isn't null and false if both are null.

Let me know if that's what we want.